### PR TITLE
feat: add bosch smoke alarm II ssiasZone command

### PR DESCRIPTION
**Pull request by vjonas**
I ran into formatting issues. It logs too many changes. Not sure how to align that, how should I do that? @Koenkk 
Potentially we could enable some formatter as Prettier to the codebase?
This should allow users to send a mqtt topic to toggle the alarm and intruder state of the smoke detector II.

### CODE REVIEW

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -2659,205 +3105,198 @@ const Cluster: {
...
             },
             boschTestTamper: {
-                ID: 0xF3,
-                parameters: [
-                    {name: 'data', type: DataType.uint8},
-                ],
+                ID: 0xf3,
+                parameters: [{name: "data", type: DataType.uint8}],
+            },
+            boschSmokeDetectorSiren: {
```
Comment (vjonas) [inline src/zcl/definition/cluster.ts:3134]: changes should be here

**Comment by Koenkk** [comment]
Could you undo the formatting changes and just add the cluster?

**Comment by vjonas** [comment]
> Could you undo the formatting changes and just add the cluster?

What would be the best way to do that? Using VSCode, I tried auto fix linting

**Comment by Koenkk** [comment]
I would revert all changes and add just the lines needed.

**Comment by vjonas** [comment]
> I would revert all changes and add just the lines needed.

Should be good now.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**